datafusion/proto/src/logical_plan/mod.rs

```diff
@@ -1641,6 +1642,7 @@ impl AsLogicalPlan for LogicalPlanNode {
                 output_url,
                 file_format,
                 copy_options,
+                partition_by: _,
```

Comment:
Note that I did not add support for partition_by in proto. We should add a follow up ticket for this. 

I don't believe this PR will break downstream systems like Ballista's handling of COPY, but it will silently ignore partition_by options until prost is updated.